### PR TITLE
[Localization] Translate party UI

### PR DESCRIPTION
**Pull request by Nolway**
## What are the changes?
Translate missing sentences and words on the party UI.

## Why am I doing these changes?
Because few sentences and words aren't translated.

## What did change?
Theses sentences will be translatable.

### Screenshots/Videos
image

The reason why I change the wording of "Cancel" on the french translation:
image

## How to test the changes?
Go into the party UI with the french language.

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/fr/party-ui-handler.ts [hunk 1]
```diff
@@ -4,8 +4,14 @@ export const partyUiHandler: SimpleTranslationEntries = {
...
+  "CANCEL": "Sortir",
   "RELEASE": "Relâcher",
   "APPLY": "Appliquer",
-  "TEACH": "Apprendre"
+  "TEACH": "Apprendre",
+  "chooseAPokemon": "Choisir un Pokémon.",
+  "doWhatWithPokemon": "Que faire avec ce Pokémon ?",
+  "selectMove": "Sélectionner une capacité.",
+  "selectItemTransfer": "Sélectionnez un objet tenu à transférer.\NUtilisez les touches < et > pour modifier la quantité.",
+  "selectPokemonSplice": "Sélectionner un autre Pokémon à séparer.",
```
Comment (Dakurei) [inline src/locales/fr/party-ui-handler.ts:15]:
```suggestion
  "selectPokemonSplice": "Sélectionnez un autre Pokémon à fusionner.",
```

**Review by Dakurei — CHANGES_REQUESTED** [review]
Some mistakes

File: src/locales/fr/party-ui-handler.ts [hunk 2]
```diff
@@ -4,8 +4,14 @@ export const partyUiHandler: SimpleTranslationEntries = {
...
-  "CANCEL": "Annuler",
+  "CANCEL": "Sortir",
   "RELEASE": "Relâcher",
   "APPLY": "Appliquer",
-  "TEACH": "Apprendre"
+  "TEACH": "Apprendre",
+  "chooseAPokemon": "Choisir un Pokémon.",
+  "doWhatWithPokemon": "Que faire avec ce Pokémon ?",
+  "selectMove": "Sélectionner une capacité.",
+  "selectItemTransfer": "Sélectionnez un objet tenu à transférer.\NUtilisez les touches < et > pour modifier la quantité.",
```
Comment (Dakurei) [inline src/locales/fr/party-ui-handler.ts:14]:
```suggestion
  "selectItemTransfer": "Sélectionnez un objet tenu à transférer.\nUtilisez les touches < et > pour modifier la quantité.",
```

File: src/locales/fr/party-ui-handler.ts [hunk 3]
```diff
@@ -4,8 +4,14 @@ export const partyUiHandler: SimpleTranslationEntries = {
...
+  "CANCEL": "Sortir",
   "RELEASE": "Relâcher",
   "APPLY": "Appliquer",
-  "TEACH": "Apprendre"
+  "TEACH": "Apprendre",
+  "chooseAPokemon": "Choisir un Pokémon.",
+  "doWhatWithPokemon": "Que faire avec ce Pokémon ?",
+  "selectMove": "Sélectionnez une capacité.",
+  "selectItemTransfer": "Sélectionnez un objet tenu à transférer.\nUtilisez les touches < et > pour modifier la quantité.",
+  "selectPokemonSplice": "Sélectionnez un autre Pokémon à séparer.",
```
Comment (Dakurei) [inline src/locales/fr/party-ui-handler.ts:15]:
```suggestion
  "selectPokemonSplice": "Sélectionnez un autre Pokémon à fusionner.",
```

**Review by Dakurei — CHANGES_REQUESTED** [review]
That remaining error

File: src/locales/fr/party-ui-handler.ts [hunk 4]
```diff
@@ -4,8 +4,14 @@ export const partyUiHandler: SimpleTranslationEntries = {
   "ALL": "Tout",
   "SEND_OUT": "Envoyer",
   "SUMMARY": "Résumé",
-  "CANCEL": "Annuler",
+  "CANCEL": "Sortir",
```
Comment (Dakurei) [inline src/locales/fr/party-ui-handler.ts:7]:
```suggestion
  "CANCEL": "Retour",
```

For the `cancelButton` I don't mind "Sortir" being used, but as far as this input is concerned, it's been "Retour" for several generations now.

Your change:
Capture d’écran du 2024-06-17 14-04-41

In official French games:
Screenshot_20240617_143137_SuperNDS

Comment (Dakurei) [inline src/locales/fr/party-ui-handler.ts:7]:
```suggestion
  "CANCEL": "Retour",
```

File: src/locales/fr/party-ui-handler.ts [hunk 5]
```diff
@@ -4,8 +4,14 @@ export const partyUiHandler: SimpleTranslationEntries = {
...
   "RELEASE": "Relâcher",
   "APPLY": "Appliquer",
-  "TEACH": "Apprendre"
+  "TEACH": "Apprendre",
+  "chooseAPokemon": "Choisir un Pokémon.",
+  "doWhatWithPokemon": "Que faire avec ce Pokémon ?",
+  "selectMove": "Sélectionnez une capacité.",
+  "selectItemTransfer": "Sélectionnez un objet tenu à transférer.\nUtilisez les touches < et > pour modifier la quantité.",
+  "selectPokemonSplice": "Sélectionnez un autre Pokémon à fusionner.",
+  "cancelButton": "Retour",
```
Comment (Dakurei) [inline src/locales/fr/party-ui-handler.ts:16]:
```suggestion
  "cancelButton": "Sortir",
```

**Review by Dakurei — CHANGES_REQUESTED** [review]
You've inverted what I told you

If you could apply the changes from Github rather than push force all your commits to each edits, it would also be simpler, and less illegible in the content of the PR

File: src/locales/ko/party-ui-handler.ts [hunk 6]
```diff
@@ -6,5 +6,11 @@ export const partyUiHandler: SimpleTranslationEntries = {
...
   "RELEASE": "놓아준다",
   "APPLY": "사용한다",
-  "TEACH": "가르친다"
+  "TEACH": "가르친다",
+  "chooseAPokemon": "Choose a Pokémon.",
+  "doWhatWithPokemon": "Do what with this Pokémon?",
+  "selectMove": "Select a move.",
+  "selectItemTransfer": "Select a held item to transfer.\nUse < and > to change the quantity.",
+  "selectPokemonSplice": "Select another Pokémon to splice.",
+  "cancelButton": "Cancel",
```
Comment (returntoice) [inline src/locales/ko/party-ui-handler.ts:15]:
```suggestion
  "chooseAPokemon": "포켓몬을 선택하세요.",
  "doWhatWithPokemon": "무엇을 하겠습니까?",
  "selectMove": "기술을 선택하세요.",
  "selectItemTransfer": "옮기고자 하는 지닌 도구를 선택하세요.\n<와 > 키를 사용하여 수량을 변경할 수 있습니다.",
  "selectPokemonSplice": "융합할 다른 포켓몬을 선택하세요.",
  "cancelButton": "그만둔다",
```

**Review by returntoice — CHANGES_REQUESTED** [review]
Korean translation

**Comment by NicusPulcis** [comment]
Italian PR has been sent

File: src/locales/es/party-ui-handler.ts [hunk 7]
```diff
@@ -6,5 +6,11 @@ export const partyUiHandler: SimpleTranslationEntries = {
...
   "RELEASE": "Release",
   "APPLY": "Apply",
-  "TEACH": "Teach"
+  "TEACH": "Teach",
+  "chooseAPokemon": "Choose a Pokémon.",
+  "doWhatWithPokemon": "Do what with this Pokémon?",
+  "selectMove": "Select a move.",
+  "selectItemTransfer": "Select a held item to transfer.\nUse < and > to change the quantity.",
+  "selectPokemonSplice": "Select another Pokémon to splice.",
+  "cancelButton": "Cancel",
```
Comment (GoldTra) [inline src/locales/es/party-ui-handler.ts:15]:
```suggestion
  "chooseAPokemon": "Elige un Pokémon.",
  "doWhatWithPokemon": "¿Que quieres hacer con el Pokémon?",
  "selectMove": "Selecciona un movimiento.",
  "selectItemTransfer": "Selecciona un objeto equipado para transferir.\nUsa < y > para cambiar la cantidad.",
  "selectPokemonSplice": "Selecciona otro Pokémon para fusionar.",
  "cancelButton": "Salir",
```

**Review by GoldTra — COMMENTED** [review]
Spanish translations

**Review by GoldTra — APPROVED** [review]

**Review by returntoice — APPROVED** [review]

**Comment by bennybroseph** [comment]
Surely we already have some of these menu options translated. Why are we retranslating Cancel?

**Comment by EnochG1** [comment]
@Nolway 
Maybe this PR is duplicated with https://github.com/pagefaultgames/pokerogue/pull/2566

**Pull request closed without merging**